packages/open-next/src/adapters/server-adapter.ts

```diff
@@ -24,20 +23,40 @@ declare global {
 
 const CACHE_BUCKET_REGION = process.env.CACHE_BUCKET_REGION;
 
+function parseS3ClientConfigFromEnv(): S3ClientConfig {
+  return {
+    region: CACHE_BUCKET_REGION,
+    logger: awsLogger,
+    maxAttempts: parseNumberFromEnv(process.env.AWS_SDK_S3_MAX_ATTEMPTS),
```

Comment:
Is the "defaults to 3" an AWS default config?

Reply:
Yessir.  Effectively `undefined` === 3.  I could change the docs to be "defaults to whatever the SDK defaults to" if that is preferred, but I think that is the direction I would take.

Reply:
Thx, that's fine, I just wanted to make sure we didn't miss the defaults.